web-admin/src/routes/-/embed/+page.svelte

```diff
@@ -25,18 +26,24 @@
 
   // Manage active resource
   let activeResource: V1ResourceName | null = null;
-  if (initialResourceName && initialResourceType) {
+  $: if (resourceName && resourceType) {
     activeResource = {
-      name: initialResourceName,
-      kind: initialResourceType,
+      name: resourceName,
+      kind: resourceType,
     };
```

Comment:
Can we do `else { activeResource = null }`, so we can avoid manually setting `activeResource=null` in `handleGoHome()`?

Reply:
Can you think of ways we can make this more clear? This feels brittle (someone could easily mistakenly make this change when modifying the code in the future).

One thing I'm noticing: the `newUrl` in `handleGoHome()` will be the URL of the last dashboard (just without the search parameters), right? That doesn't seem semantically correct.